src/Bundler/EntryPoint.php

```diff
@@ -15,27 +17,37 @@
 final class EntryPoint
 {
     private $file;
-    private $bundle_files;
-    private $vendor_files;
-    private $asset_files;
+    private $files       = [];
+    private $asset_files = [];
 
-    public function __construct(DependencyNodeInterface $file)
-    {
+    public function __construct(
+        DependencyNodeInterface $file,
+        EntryPointSplittingStrategyInterface $split_strategy = null
+    ) {
+        if ($split_strategy === null) {
+            $split_strategy = new OneOnOneSplittingStrategy();
+        }
         $this->file = $file->getFile();
 
-        // Split the input files into bundle and vendor files.
-        $this->bundle_files = [$file];
-        $this->vendor_files = [];
-        $this->asset_files  = [];
+        $result = $split_strategy->resolveChunk($file->getFile()->getName(), $file);
+        if ($result) {
```

Comment:
`null !== $result` for clarity?

Reply:
I actually removed that from a previous commit for the reason that you can't have an empty file name, so it seems to me that you shouldn't be able to split a chunk to an empty file. I could have added an error on empty string or just take the simple way out and consider null and '' as 'do not include'